src/styles.css

```
@@ -709,3 +709,83 @@ footer a:hover{text-decoration:underline}
+@media screen and (max-width: 768px) {
+
+  .nav-inner {
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    gap: 14px;
+    height: auto;
+  }
+
+  .nav-links,
+  #mobileMenu {
```

Stylelint (17.12.0)

[error] Expected id selector "`#mobileMenu`" to be kebab-case (selector-id-pattern)

(selector-id-pattern)